app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -895,14 +895,14 @@ fun MessageList(
 ) {
     val prevItemCount = remember { mutableStateOf(lazyPagingMessages.itemCount) }
     LaunchedEffect(lazyPagingMessages.itemCount) {
-        if (lazyPagingMessages.itemCount > prevItemCount.value) {
-            prevItemCount.value = lazyPagingMessages.itemCount
-
-            if (lazyListState.firstVisibleItemIndex > 0
+        if (lazyPagingMessages.itemCount > prevItemCount.value && selectedMessageId == null) {
+            if (prevItemCount.value > 0
+                && lazyListState.firstVisibleItemIndex > 0
                 && lazyListState.firstVisibleItemIndex <= MAXIMUM_SCROLLED_MESSAGES_UNTIL_AUTOSCROLL_STOPS
             ) {
```

Comment:
```suggestion
           val canScrollToLastMessage = prevItemCount.value > 0
                && lazyListState.firstVisibleItemIndex > 0
                && lazyListState.firstVisibleItemIndex <= MAXIMUM_SCROLLED_MESSAGES_UNTIL_AUTOSCROLL_STOPS
            if (canScrollToLastMessage)
```

Suggestion: we can create a var to better express the condition here, like above 👆  or maybe a better name we can come up :D

Reply:
Good idea 👍